### PR TITLE
Feat/challenge 03 loops conditionals

### PR DESCRIPTION
**Pull request by MartiCobos99**

### CODE REVIEW

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    8. Change animal to different values to make sure it works in all cases
+*/
+
+/* RESPONSE */
+
+console.log("--- Animal Checker Program ---");
+
+let animal = "dog";
+
+if (animal === "cat") {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js:18]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+/* RESPONSE */
+
+let number = 7;
+
+if (number % 2 === 0) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js:15]: ⚠️ FIX: Change the variable to test it works for different cases. Print result.

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  console.log("You are in elementary school, kid.");
+} else if (age >= 13 && age < 18) {
+  console.log("Nice, you're a teenager!");
+} else if (age >= 18) {
+  console.log("Most places consider you an adult.");
+} else {
+  console.log("What? How did this happen!?");
+}
+
+/* To fix this and get the desired output, you should use if-else if-else structure instead of multiple independent if statements.*/
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js:24]: ⚠️ FIX: needs work... 🤔

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+/* RESPONSE */
+
+// for even numbers
+for (let number = 10; number <= 40; number += 2) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:10]: These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the even ones

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  console.log(number);
+}
+
+// fot odd numbers
+for (let number = 11; number <= 39; number += 2) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:15]: These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd ones

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+/* RESPONSE */
+
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js:24]: This is the output of your script
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Half way there!
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
You made it!
All, done!
```

have you tested it before pushing? 🤔

⚠️ FIX: needs work.

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    4. After the loops is done print out: "Sucessfully logged in!"
+
+    HINT1: Be careful for an infinite loop!
+*/
+
+/* RESPONSE */
+
+let loggedIn = false;
+
+while (!loggedIn) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js:15]: This is the output of your script...
```
Incorrect login credentials
Successfully logged in!
```

...have you tested it before pushing? 🤔

⚠️ FIX: needs work.

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+/* RESPONSE */
+
+let i = 1;
+while (i <= 100) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js:25]: This is the output of your script...
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Half way there!
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
You made it!
All, done!
```

...have you tested it before pushing? 🤔

⚠️ FIX: needs work.

**Review by tictools — CHANGES_REQUESTED** [review]

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 9]
```diff
@@ -18,6 +18,10 @@ const evenOdd = function (number) {
   return result;
 };
 
+
+.
```
Comment (AGUACATE2304) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js:22]: FIX: Fatal error. Did you tested the code before upload??

Keep Working 😡

**Review by AGUACATE2304 — CHANGES_REQUESTED** [review]
Needs Works 🚀

File: challenge_03_loops-conditionals/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/README.md:1]: FIX: move this file. README should only live at project root level.

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by AGUACATE2304 — APPROVED** [review]

**Review by tictools — APPROVED** [review]

**Pull request merged**